Xamarin.Forms.Core/Color.cs

```diff
@@ -22,11 +24,14 @@ public static Color Default
 			get { return new Color(-1d, -1d, -1d, -1d, Mode.Default); }
 		}
 
-		internal bool IsDefault
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsDefault
 		{
 			get { return _mode == Mode.Default; }
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static void SetAccent(Color value) => Accent = value;
```

Comment:
Before the EditorBrowsable we would have done that because moving the method to the internals namespace as an extension was the only way to indicate to customers they shouldn't bind to the method. With the browser attribute we can leave the logic where it was originally placed. Not saying it shouldn't be an extension for other reasons but as far as IVT is concerned this change is smaller and gets the job done.